### PR TITLE
security fixes openvswich dhcp controller

### PR DESCRIPTION
**Pull request by joey00072**
dhcp-controller and openvswitch security fixes

```zsh
(base) ➜  ~
(base) ➜  ~ trivy image docker.io/platform9/openvswitch:v2.17.5-2
2023-11-09T19:07:26.443Z	INFO	Need to update DB
2023-11-09T19:07:26.443Z	INFO	DB Repository: ghcr.io/aquasecurity/trivy-db
2023-11-09T19:07:26.443Z	INFO	Downloading DB...
40.73 MiB / 40.73 MiB [---------------------------------------------------------------------------------------------------------------------------------------------------------] 100.00% 17.07 MiB p/s 2.6s
2023-11-09T19:07:29.607Z	INFO	Vulnerability scanning is enabled
2023-11-09T19:07:29.607Z	INFO	Secret scanning is enabled
2023-11-09T19:07:29.607Z	INFO	If your scanning is slow, please try '--scanners vuln' to disable secret scanning
2023-11-09T19:07:29.607Z	INFO	Please see also https://aquasecurity.github.io/trivy/v0.45/docs/scanner/secret/#recommendation for faster secret detection
2023-11-09T19:07:29.628Z	INFO	Detected OS: alpine
2023-11-09T19:07:29.629Z	INFO	Detecting Alpine vulnerabilities...
2023-11-09T19:07:29.644Z	INFO	Number of language-specific files: 1
2023-11-09T19:07:29.644Z	INFO	Detecting python-pkg vulnerabilities...

docker.io/platform9/openvswitch:v2.17.5-2 (alpine 3.16.7)

Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)

(base) ➜  ~ trivy image docker.io/platform9/openvswitch:v2.17.5-2
2023-11-09T19:07:49.649Z	INFO	Vulnerability scanning is enabled
2023-11-09T19:07:49.649Z	INFO	Secret scanning is enabled
2023-11-09T19:07:49.649Z	INFO	If your scanning is slow, please try '--scanners vuln' to disable secret scanning
2023-11-09T19:07:49.649Z	INFO	Please see also https://aquasecurity.github.io/trivy/v0.45/docs/scanner/secret/#recommendation for faster secret detection
2023-11-09T19:07:49.666Z	INFO	Detected OS: alpine
2023-11-09T19:07:49.667Z	INFO	Detecting Alpine vulnerabilities...
2023-11-09T19:07:49.681Z	INFO	Number of language-specific files: 1
2023-11-09T19:07:49.681Z	INFO	Detecting python-pkg vulnerabilities...

docker.io/platform9/openvswitch:v2.17.5-2 (alpine 3.16.7)

Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

### CODE REVIEW

File: controllers/networkplugins_controller.go [hunk 1]
```diff
@@ -58,11 +58,11 @@ const (
 	WhereaboutsImage        = "docker.io/platform9/whereabouts:v0.6-pmk-2871011"
 	SriovCniImage           = "docker.io/platform9/sriov-cni:v2.6.2-pmk-2877848"
 	SriovDpImage            = "docker.io/platform9/sriov-network-device-plugin:v3.3.2-pmk-2877839"
-	OvsImage                = "docker.io/platform9/openvswitch:v2.17.5-1"
+	OvsImage                = "docker.io/platform9/openvswitch:v2.17.5-2"
```
Comment (cruizen) [inline controllers/networkplugins_controller.go:61]: Where is this image built? What is the source of this image?

Comment (joey00072) [reply controllers/networkplugins_controller.go:61]: there is no tc build for this,
we have to build this locally and push it
this is located in hostplumber/pkg/ovs_docker

Comment (joey00072) [reply controllers/networkplugins_controller.go:61]: Yeah sure

Comment (cruizen) [reply controllers/networkplugins_controller.go:61]: @joey00072 Do we have a JIRA issue for this now? We still see critical vulnerabilities reported in `docker.io/platform9/openvswitch`

**Pull request merged**